src/utils/normalizer.test.ts

```diff
@@ -0,0 +1,136 @@
+import { normalizeEIP712TypedMessageData } from './normalizer';
+
+describe('normalizeEIP712TypedMessageData', () => {
+  const messageData = {
```

Comment:
Should we instead use a file level constant with uppercase and underscores?

Reply:
Done